### PR TITLE
allows to upload private packages

### PR DESCRIPTION
**Pull request by j0hn**
Adds `--private` option to create a package as private directly from the command line.

### CODE REVIEW

File: binstar_client/commands/upload.py [hunk 1]
```diff
@@ -293,6 +295,7 @@ def add_parser(subparsers):
     mgroup.add_argument('-t', '--package-type', help='Set the package type, defaults to autodetect')
     mgroup.add_argument('-d', '--description', help='description of the file(s)')
     mgroup.add_argument('--thumbnail', help='Notebook\'s thumbnail image')
+    mgroup.add_argument('--private', help="Create the package as private access", action='store_true')
```
Comment (abarto) [inline binstar_client/commands/upload.py]: The help message should be "Create the package with private access"

Comment (j0hn) [inline binstar_client/commands/upload.py]: thanks

**Comment by goanpeca** [comment]
@j0hn any chance of adding a test :-p ?

**Comment by j0hn** [comment]
@goanpeca yep, you're right, i just added a few, thanks!

**Pull request merged**